src/main/java/org/broadinstitute/consent/http/db/ConsentDAO.java

```diff
@@ -31,7 +31,7 @@
     @SqlQuery("SELECT c.name " +
             "FROM consents c INNER JOIN consentassociations cs ON c.consentId = cs.consentId "+
             "WHERE cs.dataSetId = :datasetId")
-    String findConsentNameFromDatasetID(@Bind("datasetId") String datasetId);
+    String findConsentNameFromDatasetID(@Bind("datasetId") Integer datasetId);
```

Comment:
What is the purpose of making the dataset Id an Integer instead of a String?

Reply:
`datasetId` is actually an integer in the database. This was a pre-existing bug that we never saw and didn't break in mysql. In postgres, it breaks :-(

Reply:
That makes sense! Thank you!